src/ResourceManagement/Compute/Domain/IDisk.cs

```diff
@@ -32,6 +32,11 @@ public interface IDisk :
         /// </summary>
         Models.OperatingSystemTypes? OSType { get; }
 
+        /// <summary>
+        /// Gets the hypervisor generation of the Virtual Machine.
```

Comment:
Should be disk?

Reply:
The description is provided by service team within DiskInner. The hypervisor generation is related to virtual machine while it is applicable to OS disk only.